openvdb/openvdb/CMakeLists.txt

```diff
@@ -718,29 +718,58 @@ endif()
 
 # Installation
 
-if(OPENVDB_CORE_STATIC)
-  install(TARGETS openvdb_static
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  )
-endif()
+if(SKBUILD)
```

Comment:
Wondering if there's a cleaner way to handle this kind of case without all of this conditional branching. What about using an INSTALL_PREFIX and including the forward slash in the prefix (although potentially needs thought for multi-platforms):
```
if (SKBUILD)
  set(INSTALL_PREFIX "openvdb/")
else
  set(INSTALL_PREFIX "")
endif()

install(TARGETS openvdb_static
  RUNTIME_DESTINATION ${INSTALL_PREFIX}${CMAKE_INSTALL_BINDIR}
)
```
Ultimately, it would be nice to use less `if (SKBUILD)` logic if possible?

Reply:
Yeah, I'm not a huge fan of all the `SKBUILD` logic either. I'm happy to take another look.

In an ideal world, it'd be possible to separate them entirely, but in practice Python packaging has very limited support for _two_ modules in a single package. Thus, we need to delegate a lot of the pip install location logic to CMake since Python just doesn't provide us with the expressiveness to do so.

Reply:
Thanks for the suggestions @danrbailey ! I've consolidated the conditional branching significantly. Please take another look and let me know what you think.